x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_expression.tsx

```diff
@@ -142,7 +150,15 @@ export function XYChartReportable(props: XYChartRenderProps) {
   );
 }
 
-export function XYChart({ data, args, formatFactory, timeZone }: XYChartRenderProps) {
+export function XYChart({
+  data,
+  args,
+  formatFactory,
+  timeZone,
+  handlers,
+}: XYChartRenderProps & {
+  handlers: IInterpreterRenderHandlers;
+}) {
```

Comment:
@ppisljar I can't tell if it's better to do `handlers.event(...)` vs `npStart.plugins.uiActions.executeTriggerActions('VALUE_CLICK_TRIGGER',`, can you clarify the expected use?

Reply:
`handlers.event` is better because it avoids direct, legacy access of `npStart.plugins.uiActions`.

Reply:
@stacey-gammon could you clarify what API on `handlers` can be used to `executeTriggerActions`? I am not finding any API that could used to get access to `uiActions` via handlers?

Reply:
I thought the plan was to expose uiActions on handles, either from work Peter was doing, or Wylie. Afaik it is not exposed on handler on master, but I haven't kept up with the status.